README.rst

```diff
@@ -598,6 +598,20 @@ Choose from the list of available rules:
     'SuppressWarnings', 'noinspection', 'package_version', 'enduml',
     'startuml', 'fix', 'FIXME', 'fixme', 'override']``
 
+* **echo_tag_syntax** [@PhpCsFixer]
+
+  Replaces short-echo ``<?=`` with long format ``<?php echo``/``<?php print``
+  syntax, or vice-versa.
+
+  Configuration options:
+
+  - ``format`` (``'long'``, ``'short'``): the desired language construct; defaults to
+    ``'long'``
+  - ``long_function`` (``'echo'``, ``'print'``): the function to be used to expand the
+    short echo tags; defaults to ``'echo'``
+  - ``short_always`` (``bool``): always render short-echo tags even in case of
```

Comment:
`handle_complex_code`? `short_always` looks meaningless.

Reply:
`handle_complex_code` with a boolean value? I don't think it's very clear...

Reply:
What about `shorten_simple_statements_only`, defaulting to `true`? A bit verbose but very explicit :)

Reply:
Done in https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4493/commits/034546b04580b786f795945c9f191dfc9cb8e218